commit 	593de6855a4f4dc26cface3e96de8889f90cb4bb
Author: Heiko Becker
Date:	Mon Oct 20 12:46:34 2014 +0000

    Fix build with libressl

    libressl removed the SSL Compression functionality (which might
    be considered insecure) and thus also compress_meth.
    SSL_SESSION_get_compress_id is just a stub in libressl which always
    returns 0 and in openssl it returns compress_meth.

    REVIEW: 121107

--- plugins/qca-ossl/qca-ossl.cpp
+++ plugins/qca-ossl/qca-ossl.cpp
@@ -5805,7 +5805,7 @@
 	{
 		SessionInfo sessInfo;
 
-		sessInfo.isCompressed = (0 != ssl->session->compress_meth);
+		sessInfo.isCompressed = (0 != SSL_SESSION_get_compress_id(ssl->session));
 
 		if (ssl->version == TLS1_VERSION)
 			sessInfo.version = TLS::TLS_v1;

